### DISCUSSION TITLE
Is it possible to set spacing after/before paragraph through CSS?

### DISCUSSION THREAD

**Discussion by Serenade600** [Zettlr/Zettlr#2904] [Q&A] [answered]
Even if a code editor like codemirror may not has a concept of "paragraph", does Zettlr could set spacing after/before paragraph appearing on screen like Word?

**Comment by nathanlesage** [comment, marked as answer]
No. The concept of a "paragraph" is defined in Markdown as an empty line between two lines, and that already adds enough spacing. Single newlines are normally stripped on export, so it's considered best practice to always delimit what you want to be paragraphs with two newlines.
